src/core.jl

```diff
@@ -51,7 +51,7 @@ struct Cosmology
     ks::Array
     pk0::Array
     dlogk
-    lplk::AbstractInterpolation
+    primordial_lPk::AbstractInterpolation
```

Comment:
I'm not sure I understand what you're calling "primordial_lPk" here. I think it is the linear power spectrum at z=0, right? If so, that's not the primordial power spectrum, so I'd call it `linear_lPk_z0`, for instance.

Although, not sure if you really need this, if you're already storing the linear power spectrum at all redshifts as well.